askstylo/superset
# Drilldown

### SUMMARY

Add a "drilldown" feature to echarts:pie and nvd3:dist_bar.  Majority of design taken from https://github.com/apache/superset/pull/14688.  

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF

See video on the PR linked above and https://askstylo.slack.com/files/U028GAL04PP/F02EQEDG7AA/_dev__superset_-_google_chrome_2021-09-18_18-02-05.mp4

### TESTING INSTRUCTIONS

Create a pie or bar chart with multiple "group by" or "series" fields which form a coherent hierarchy.  Click the "enable drill down" control in the chart setup control panel.  On graph refresh, the chart should only show entries from the top level in the hierarchy.  Then click one of the resulting series to view the next deeper level in the hierarchy, etc..  

### ADDITIONAL INFORMATION


- [x] Required feature flags: DASHBOARD_DRILL_DOWN
